pyHS100/cli.py

```diff
@@ -9,16 +9,18 @@
           sys.version_info)
     sys.exit(1)
 
-from pyHS100 import SmartPlug, TPLinkSmartHomeProtocol  # noqa: E402
+from pyHS100 import (SmartDevice, SmartPlug, SmartBulb,
+                     TPLinkSmartHomeProtocol)  # noqa: E402
 
-pass_dev = click.make_pass_decorator(SmartPlug)
+pass_dev = click.make_pass_decorator(SmartDevice)
 
 
 @click.group(invoke_without_command=True)
 @click.option('--ip', envvar="PYHS100_IP", required=False)
 @click.option('--debug/--normal', default=False)
+@click.option('--bulb', default=False, is_flag=True)
 @click.pass_context
-def cli(ctx, ip, debug):
+def cli(ctx, ip, debug, bulb):
     """A cli tool for controlling TP-Link smart home plugs."""
```

Comment:
Ungh, sorry, forgot to fix this. Will try to remember to do it next time there will be code cleanups.